k8s/crd-samples/pool-crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorpools.openebs.io
+  namespace: default
+spec:
+  serviceAccountName: openebs-maya-operator
```

Comment:
why do these CRDs require a serviceAccountName?

Reply:
we need to enclose the list of crds under a particular serviceAccountName with their permissions.

Reply:
I don't see the need for this - for example the storagepool, storagepoolclaims didnot have a serviceAccountName associated with it. Why do we need this for CStorPool?